source/speech/speech.py

```diff
@@ -1743,6 +1743,7 @@ def getControlFieldSpeech(  # noqa: C901
 			config.conf["presentation"]["reportObjectDescriptions"]
 			and (
 				reason == OutputReason.FOCUS
+				# alwaysReportDescription isn't set anywhere or respected by braille or when aria-description is the source
 				or attrs.get('alwaysReportDescription', False)

```

Comment:
This should have been a review comment - `alwaysReportDescription` isn't set anywhere or respected by braille or used when aria-description is the source. I'm not sure if this is an issue.

```suggestion
				or attrs.get('alwaysReportDescription', False)
```